flink-runtime/src/test/java/org/apache/flink/runtime/metrics/util/MetricUtilsTest.java

```diff
@@ -59,6 +68,58 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(MetricUtilsTest.class);
```

Comment:
```suggestion
```
That's a bit unrelated to your change but we could actually remove the `LOG` here as it's not used anywhere.

Reply:
@XComp Will remove it in fixup commit.